requirements/docs.txt

```diff
@@ -10,3 +10,5 @@ matplotlib>=3.0.1
 dask[array]>=0.15.0
 # cloudpickle is necessary to provide the 'processes' scheduler for dask
 cloudpickle>=0.2.1
+pandas
+seaborn
```

Comment:
Please, specify some minimal version here, similar to another PR of yours.

Reply:
Fixed in 7f9a303c5d6a61f8611de4c3d67caa59c7d0e573. Too bad the NEP 29 script didn't work out of the box for `pandas`, because it has old versions such as `'0.2b1'` and `'2beta'`!